github/repos_hooks.go

```diff
@@ -98,6 +98,31 @@ type createHookRequest struct {
 	Active *bool                  `json:"active,omitempty"`
 }
 
+// PubSubHubBubRequest represents the request to be send
+// for the GitHub's publish/subscribe hub protocol.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/hooks/#subscribing
+type PubSubHubBubRequest struct {
+	HubMode     string `json:"hub_mode,omitempty"`
+	HubTopic    Topic  `json:"hub_topic,omitempty"`
+	HubCallback string `json:"hub_callback,omitempty"`
+	HubSecret   string `json:"hub_secret,omitempty"`
+}
+
+// Topic represents GitHub repository and its particular event.
+type Topic struct {
+	Owner string `json:"owner,omitempty"`
+	Repo  string `json:"repo,omitempty"`
+	Event string `json:"events,omitempty"`
+}
+
+type pubSubHubBubPayload struct {
```

Comment:
`Bub` => `bub`

Reply:
@gmlewis can decide if this is a change that makes sense.

If the `CreatePubSubHubBubRequest` request/opts parameter gets replaced with `owner, repo, event string` (matching the way owner/repo args are used by other functions in go-github) then `pubSubHubBubPayload` could get turned into `PubSubHubbubOpts` as was suggested elsewhere by gmlewis, and `PubSubHubbubRequest` could be removed entirely.